docs/schema.md

```diff
@@ -1104,4 +1104,98 @@ Representation of a GCP [Organization](https://cloud.google.com/resource-manager
 | lastupdated |  Timestamp of the last time the node was updated | 
 | id | The name of the GCP Organization, e.g. "organizations/1234" |
 | displayname | The "friendly name", e.g. "My Company"
-| lifecyclestate | The organization's current lifecycle state. Assigned by the server.  See the [official docs](https://cloud.google.com/resource-manager/reference/rest/v1/organizations#LifecycleState). 
\ No newline at end of file
+| lifecyclestate | The organization's current lifecycle state. Assigned by the server.  See the [official docs](https://cloud.google.com/resource-manager/reference/rest/v1/organizations#LifecycleState).
```

Comment:
it's starting to get pretty big. Should we consider breaking the doc into core pieces?